docs/guides/creating-a-reproduction.md

```diff
@@ -22,6 +22,18 @@ just delete and recreate the environment. It's trivial to set up:
 
 -   Activate the environment with:
 
+
+=== ":fontawesome-brands-windows: Windows"
+
+    ``` sh
+    . venv/Scripts/activate # (1)!
+    ```
+    
+    1.  Your terminal should now print `(venv)` before the prompt, which is
+        how you know that you are inside an environment.
+        
+=== ":material-linux: Linux"
```

Comment:
Please create separate tabs for macOS, Windows and Linux, as we did here:
https://squidfunk.github.io/mkdocs-material/setup/dependencies/image-processing/#cairo-graphics

Source is here:
https://github.com/squidfunk/mkdocs-material/blob/master/docs/setup/dependencies/image-processing.md

Reply:
I already did it, but should I cahnge Linux to Linux, macOS ?

Reply:
Yes, please stick to the format mentioned in the links ☺️